packages/experiments/src/components/Button/DefaultButton.tsx

```diff
@@ -3,7 +3,7 @@ import { Button } from './Button';
 import { ButtonVariantsType } from './ButtonVariants.types';
 
 export const DefaultButton: ButtonVariantsType = props => {
-  const { text, ...rest } = props;
+  const { text, iconProps, ...rest } = props;
```

Comment:
I didn't catch this in the other PR but maybe text is another prop that could be deprecated in master and removed in Fabric 7.